src/services/chains/index.ts

```
@@ -114,3 +137,29 @@ export function getAppropriateNetworkId(params: {
+    abi,
+    provider,
+  );
+  const result: string | undefined = undefined;
+  for (const log of receipt.logs) {
+    try {
+      const parsedLog = contract.interface.parseLog(log);
+      if (parsedLog.name === 'SafeReceived') {
+        return log.address; //returns the projectAddress receiving the amount
+      }
+    } catch (error) {
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)